src/screens/Hooks/GroupACLHooks.res

```diff
@@ -96,6 +100,18 @@ let useUserGroupACLHook = () => {
     | (NoAccess, NoAccess) => NoAccess
     | (_, _) => Access
     }
+  let userHasBothGroupsAccess = (group1, group2) => {
```

Comment:
It's not scalable. Can we accept an array and try to check that Array all check something property has Access and return true something.  Try this if possible.